Documentation/general/tool-nuget-package-format.md

```diff
@@ -0,0 +1,26 @@
+Tool NuGet package format
+-------------------------------------------
+
+Use `dotnet pack` with `PackAsTool` property is the most straight forward way. However, if your build process is highly customized and `dotnet publish` cannot create a proper distribution for you. You can create tool NuGet package using nuspec and explicitly place assets in NuGet package under the following rules.

```

Comment:
The most straightforward way to create a .NET tool package is to run `dotnet pack` with `PackAsTool` property set in the project file. However, if your build process is highly customized, `dotnet publish` may not create the right package for you. In this case, you can create a NuGet package for your tool using a *nuspec* file and explicitly placing assets into the NuGet package following these rules.

Reply:
I just used this version